assistant/build.gradle

```diff
@@ -60,7 +60,7 @@ checkstyle {
 dependencies {
     compile project(':common')
     testCompile project(':common').sourceSets.test.output
-    compile 'com.ibm.cloud:sdk-core:7.0.0'
+    compile 'com.ibm.cloud:sdk-core:7.1.0'
```

Comment:
If you end up using the latest version of the generator to generate the upcoming Watson sdk release, you'll need to upgrade to the new major version of the java core (8.0.0) after that is tagged (should be soon).   Just a heads up.